filebeat/filebeat.full.yml

```diff
@@ -510,6 +510,14 @@ output.elasticsearch:
   # Path to the Elasticsearch 2.x version of the template file.
   #template.versions.2x.path: "${path.config}/filebeat.template-es2x.json"
 
+  # If set to true, filebeat checks the Elasticsearch version at connect time, and if it
+  # is 5.x, it loads the file specified by the template.versions.5x.path setting. The
+  # default is true.
+  #template.versions.5x.enabled: true
+
+  # Path to the Elasticsearch 5.x version of the template file.
+  #template.versions.5x.path: "${path.config}/filebeat.template-es5x.json"
```

Comment:
Yep, I see the issue, had my doubts when setting it, perhaps the best thing we can do is switching to template-es6x and keep template.json as the default. Then deprecate this in 6.0 and move to the path based solution? Doing it now would break config compatibility